website/content/docs/lifecycle/build.mdx

```diff
@@ -232,54 +232,71 @@ the build/registry phases are one and the same.
 ### Private Registries
 
 When using the docker plugin to push images to private container registries, the
-registry authentication is configured "out of band" from Waypoint. For example
-your `waypoint.hcl` may have a `build` stanza with a `registry` section pointing
-to a private registry:
+registry authentication may either be configured explicitly or "out of band"
+from Waypoint.
+
+_Explicit Authentication_
```

Comment:
You can use `h4`s for this if you want: `####`

Reply:
Thanks, addressed this.